src/Heading/test/Heading.test.js

```diff
@@ -26,3 +33,43 @@ describe('<Heading1 />', () => {
     )
   })
 })
+describe('<Heading2 />', () => {
```

Comment:
🥳

Reply:
After the fact I found an easier way to write the CSS. I rolled it out as 4.4.2. https://github.com/adobe/parliament-ui-components/commit/0585481c4c32453f19f62ea98313f90645676a02